### PR TITLE
Fix a bug in the send_local_online_presence_to module API

### PR DESCRIPTION
**Pull request by clokep**
Reported at https://matrix.to/#/!vcyiEtMVHIhWXcJAfl:sw1v.org/$mG_v0O-a887V8k7WVH8VSA9VIagKQibmnNHew74k2PI?via=matrix.org&via=element.io&via=beeper.com

Part of #14809

### CODE REVIEW

File: changelog.d/14880.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix a bug when using the `send_local_online_presence_to` module API.
```
Comment (clokep) [inline changelog.d/14880.bugfix:1]: I'm not really sure of the impact of this bug -- sorry for the weak changelog. Any suggestions appreciated!

**Review by H-Shay — APPROVED** [review]
This seems pretty straight-forward to me, my only question is if it needs a Github issue filed (even though it's after the fact?)

**Comment by clokep** [comment]
> is if it needs a Github issue filed (even though it's after the fact?)

I don't think we usually do that -- I think it is fine without one.

**Comment by H-Shay** [comment]
Sounds good to me then!

**Comment by clokep** [comment]
Looks like @DMRobertson Already made a similar change to the `StrCollection` type!

Thanks for the review! 🎉

**Pull request merged**